tests/testing_data/validator/valid_data/actions.yml

```
@@ -0,0 +1,25 @@
+- action_name: action_say_goodbye
+  content_type: json
+  headers: []
+  http_url: https://jsonplaceholder.typicode.com/posts/1
+  params_list: []
+  request_method: GET
+  response:
+    dispatch: true
+    dispatch_type: text
+    evaluation_type: expression
+    value: ${data}
+  set_slots: []
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)